Section_II/competition_and_trophies.tex

```diff
@@ -659,28 +745,70 @@
 
 \bigskip
 
-Certificates are awarded to the teams second and third in the technical challenges,
-to the team\removed{s} ranked second \removed{and third} in the Best Humanoid list
+In a physical competition, certificates are awarded to the teams second and third in the technical challenges,
+to the team ranked second in the Best Humanoid list
 and a 'Best Player' certificate to the three most highly ranked Drop-in players.
 A team can only receive a 'Best Player' certificate if their total amount of
 points is greater than 0.
 
 \bigskip
 
-The Best Humanoid Award is awarded to the team ranked first in the Best Humanoid list,
+In a physical cometption, the Best Humanoid Award is awarded to the team ranked first in the Best Humanoid list,
 it can either be a trophy or a certificate.
 
 \bigskip
 
+
 {\bfseries Conflict Resolution}
 
 \headlinebox
 
-It is the responsibility of the team leader to inspect the other team's robot an hour in advance of a game. Any concern regarding the rule compliance of any of the robots, including the amount, size and colour of the team markers, must be brought to the attention of the referee an hour in advance of the game. If the referee is unavailable, they have to be brought to the attention of the Technical Committee instead.
-
-\bigskip
+It is the responsibility of the team leader to inspect the other team's
+robots an hour (physical competition) or six hours (virtual competition) in advance of a game.
```

Comment:
does not really match with the 12 hour guarantee we gave earlier

Reply:
Agreed, changed to twelve hours